Fix formatting in a couple docs (#47523)
CleverRaven/Cataclysm-DDA: doc/COMPILING/COMPILING-CMAKE.md

# Build Options

> A full list of options supported by CMake, you may either run the `ccmake` or `cmake-gui` front-ends, or run `cmake` and open the generated `CMakeCache.txt` from the build directory in a text editor.

Reviewer: ```suggestion
For a full list of options supported by CMake, you may either run the `ccmake` or `cmake-gui` front-ends, or run `cmake` and open the generated `CMakeCache.txt` from the build directory in a text editor.
```

```
$ cmake -DOPTION_NAME1=option_value1 [-DOPTION_NAME2=option_value2 [...]]
```

## CMake specific options

* `CMAKE_BUILD_TYPE=<build type>`

  Selects a specific build configuration when compiling. `release` produces the default, optimized (`-Os`) build for regular use. `debug` produces a slower and larger unoptimized (`-O0`) build with full debug symbols, which is often needed for obtaining detailed backtraces when reporting bugs.

  **NOTE**: By default, CMake will produce `debug` builds unless a different configuration option is passed in the command line.

* `CMAKE_INSTALL_PREFIX=<full path>`

  Installation prefix for binaries, resources, and documentation files.

## CataclysmDDA specific options

 * `CURSES=<boolean>`: Build curses version.
 * `TILES=<boolean>`: Build graphical tileset version.
 * `SOUND=<boolean>`: Support for in-game sounds & music.
 * `USE_HOME_DIR=<boolean>`: Use user's home directory for save files.
 * `LOCALIZE=<boolean>`: Support for language localizations. Also enable UTF support.
 * `LANGUAGES=<str>`: Compile localization files for specified languages. Example:

   ```
   -DLANGUAGES="cs;de;el;es_AR;es_ES"
   ```

>    Note that language files are only compiled automatically when building the `RELEASE` build type. For other build types, you need to add the `translations_compile` target to the `make` command, for example `make all translations_compile`.

Reviewer: ```suggestion
   Note that language files are only compiled automatically when building the `RELEASE` build type. For other build types, you need to add the `translations_compile` target to the `make` command: for example `make all translations_compile`.
```

>    Special note for MinGW: due to a [libintl bug](https://savannah.gnu.org/bugs/index.php?58006), using English without a `.mo` file would cause significant slow down on MinGW targets.  In such case you can compile a `.mo` file for English by adding `en` to `-DLANGUAGES`.  If `-DLANGUAGES` is not specified, it also compiles a `.mo` file for English in addition to other languages.

Reviewer: ```suggestion
   Special note for MinGW: Due to a [libintl bug](https://savannah.gnu.org/bugs/index.php?58006), using English without a `.mo` file causes significant slowdown on MinGW targets.  In such cases, you can compile a `.mo` file for English by adding `en` to `-DLANGUAGES`.  If `-DLANGUAGES` is not specified, it also compiles a `.mo` file for English in addition to other languages.
```
The second sentence is a bit unclear; @BrettDong?

Reviewer 2: CC @Qrox

Reviewer 3: The second sentence says that you can compile `en.mo` by adding `en` to the list of languages that follows `-DLANGUAGES`. For example `-DLANGUAGES="cs;de;el;es_AR;es_ES"` (taken from the example above) becomes `-DLANGUAGES="cs;de;el;es_AR;es_ES;en"`.

Author: I've put in a slight rephrasing to make this paragraph easier to understand; hope it clears things up.
